tests/integration/README.md

```diff
@@ -71,7 +71,7 @@ function teardown() {
 }
 
 @test "this is a simple test" {
-  runc start containerid
+  runc run containerid
   # "run" automatically populates $status, $output and $lines.
```

Comment:
I just realised that I didn't update this comment. Just say "the runc macro" instead of "run".